data/src/main/java/com/example/data/db/tables/LastChosenCitiesEntity.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.data.db.tables
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "last_chosen_cities_entity")
+class LastChosenCitiesEntity(
+    @PrimaryKey(autoGenerate = true)
+    val id: Int = 0,
```

Comment:
I thought you planned to use timestamp for sorting. Ok, let's use separate field for id.

Reply:
using timestamp does not make sense because I can using regular id. I changed algorithm of delitting first line from table on this `@Query("DELETE FROM last_chosen_cities_entity WHERE id = (SELECT id FROM last_chosen_cities_entity ORDER BY id LIMIT 1)")`